tests/x11/rstudio_server.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Basic test of RStudio Server
+# Maintainer: Dan Čermák <dcermak@suse.com>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use rstudio;
+
+sub run() {
+    mouse_hide(1);
+    ensure_installed('rstudio-server');
+
+    x11_start_program('xterm');
+    become_root();
+    assert_script_run("systemctl start rstudio-server");
+    wait_still_screen(1);
+    send_key("alt-f4");
+
+    # open R Studio in the browser and don't verify that Firefox actually
+    # started via a needle
+    x11_start_program("firefox http://localhost:8787", valid => 0);
+    wait_still_screen;
```

Comment:
that's a waste of time. It should be enough in the next step to just look for the needle area that you want to click.